ansible/projects/ovn-kubernetes/deploy-objects.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Deploy {{ deployment_object }}
+  k8s:
+    src: "{{ ovn_kubernetes_dir.path }}/dist/yaml/{{ deployment_object }}.yaml"
+    state: present
+    kubeconfig: "{{ kubeconfig }}"
+    context: "{{ k8s_context }}"
+- name: Get {{ deployment_object }} pods list
+  k8s_info:
+    kind: Pod
+    label_selectors:
+      - name = {{ deployment_object }}
```

Comment:
curious why not use the `name` field instead of the `label selector` field?

Reply:
The pod of deployment or daemonset will have the name of it and a suffix e.g. "ovnkube-node-<suffix>" but all the pods have a label selector with the name and the name of its deployment or daemonset